src/sentry/models/files/control_fileblob.py

```diff
@@ -12,3 +13,21 @@ class Meta:
 
     FILE_BLOB_OWNER_MODEL = ControlFileBlobOwner
     DELETE_FILE_TASK = delete_file_control
+
+    # TODO(hybrid-cloud): This is a temporary measure to allow concurrent production deployments of filestore
+    # We override the behavior of only the control silo file storage implementation to use
+    # the new control instance while production runs in monolith mode
+    @classmethod
+    def _storage_config(cls):
+        from sentry import options as options_store
+
+        config = None
+        try:
+            # If these options exist, use them. Otherwise fallback to default behavior
+            config = {
+                "backend": options_store.get("filestore.control.backend"),
+                "options": options_store.get("filestore.control.options"),
```

Comment:
`options_store.get("filestore.control.backend")` might return an empty/default value if the option is registered, but not set, thus the return config will be invalid.
I think we should at least check that "backend" is not empty.